Core.ApplicationServices/UserService.cs

```diff
@@ -306,31 +306,58 @@ public void Dispose()
                 .Transform(Result<IQueryable<User>, OperationError>.Success);
         }
 
-        public Maybe<OperationError> DeleteUserFromKitos(Guid userUuid)
+        public Maybe<OperationError> InitiateUserDeletion(Guid userUuid, int? scopedToOrganizationId = null)
         {
-            using var transaction = _transactionManager.Begin();
-
             var user = _userRepository.AsQueryable().ByUuid(userUuid);
             if (user == null)
                 return new OperationError(OperationFailure.NotFound);
             if(_organizationalUserContext.UserId == user.Id)
                 return new OperationError("You cannot delete a user you are currently logged in as", OperationFailure.Forbidden);
 
+            if (scopedToOrganizationId.HasValue && scopedToOrganizationId.Value != 0)
```

Comment:
why do you reason about it being 0?. this code should not know if 0 is a valid id.. if it is not null then it is defined by the caller hence it will mean "deletefromorg"